packages/x-charts/src/internals/stackSeries.ts

```diff
@@ -57,11 +55,11 @@ export const StackOrder: {
    * Reverse of the given series order [n - 1, n - 2, … 0] where n is the number of elements in series. Thus, the stack order is given by the reverse of the key accessor.
    */
   reverse: d3StackOrderReverse,
+} satisfies {
+  [key in StackOrderType]: (series: Series<any, any>) => number[];
 };
```

Comment:
Moving it to satisfies ensures that we are thorough with these types and are actually providing all the expected keys. 

This means that removing one of the keys or adding an unexpected key will not generate type errors